src/GeniusYield/Test/Privnet/Examples/Gift.hs

```diff
@@ -656,19 +645,3 @@ checkCollateral inputValue returnValue totalCollateralLovelace txFee collPer =
   && balanceLovelace>= ceiling (txFee * collPer % 100)  -- Api checks via `balanceLovelace * 100 >= txFee * collPer` which IMO works as `balanceLovelace` is an integer & 100 but in general `c >= ceil (a / b)` is not equivalent to `c * b >= a`.
   && inputValue == returnValue <> valueFromLovelace totalCollateralLovelace
   where (balanceLovelace, balanceOther) = valueSplitAda $ inputValue `valueMinus` returnValue
-
-resolveRefScript :: Either GYTxOutRef GYTxBody -> Some GYScript -> GYTxMonadIO GYTxOutRef
```

Comment:
Adding and resolving is combined into one. See `GeniusYield.Test.Utils.addRefScript`